requirements.txt

```diff
@@ -8,7 +8,8 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+requests>=2.9.1
 
 # Python3 compat
 future>=0.15.2
-configparser>=3.3.0; python_version < '3'
+#configparser>=3.3.0; python_version < '3'
```

Comment:
To be mor precise:

$ rpm -qf /usr/lib/python2.7/site-packages/configparser/__init__.py
python-future-0.15.2-1.mga5
$ rpm -qf /usr/lib64/python2.7/ConfigParser.py
python-2.7.9-2.mga5

So do we really need a separate module or is it part of python ?